scripts/tools/tools-schema.json

```diff
@@ -0,0 +1,82 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "JSON Schema for AsyncAPI tool discovery file.", 
+	"type": "object",
+	"required": [
+		"title",
+		"description",
+		"maintainers",
+		"links",
+		"filters"
+	],
+	"properties": {
+		"title": {
+			"type": "string",
+            "description": "Human-readable name of the tool that will be visible to people in the list of tools",
+            "examples" : [ "AsyncAPI Generator", "Cupid" ]  
+		},
+		"description": {
+			"type": "string"
+		},
+		"maintainers": {
+			"type": "array",
+			"items":{
+				"type": "string"
+			}
+
+		},
+		"links": {
+			"type": "object",
+			"required": [
+				"websiteUrl",
+				"docsUrl",
+				"iconUrl"
+			],
+			"properties": {
+				"websiteUrl": {
+					"type": "string"
+				},
+				"docsUrl": {
+					"type": "string"
+				},
+				"iconUrl": {
+					"type": "string"
+				}
+			}
+		}
+,
+		"filters": {
+			"type": "object",
+			"required": [
+				"language",
+				"technology",
+				"categories",
+				"hasCommercial"
```

Comment:
If you have defined default value for `hasCommercial`, that's field can be optional

 ```suggestion
```

Reply:
I think we should make this field optional or provide a default value. In case, anyone forgets to add a `false` value to it, an open source tool will be considered as a paid one. So, from my point of view, we should remove the default value of this field and  make this as required. What's your take @derberg?

Reply:
If tool has commercial "support" people shouldn't forgot about this and if yes, will add this as update to `.asyncapi-tool` -  so `hasCommercial` should has `false` as default value in my opinion. >90% cases will be open source tools  so `hasCommercial: false` has big sense.

Reply:
yes, default false definitely, only few will have "commercial"